docs/contributing/frontend.md

```diff
@@ -2,6 +2,15 @@
 
 This is a guide to help the Coder community and also Coder members contribute to our UI. It is ongoing work but we hope it provides some useful information to get started. If you have any questions or need help, please send us a message on our [Discord server](https://discord.com/invite/coder). We'll be happy to help you.
 
+## Running the UI
+
+You can run the UI and access the dashboard in two ways:
+
+- Build the UI pointing to an external Coder server: `CODER_HOST=https://mycoder.com yarn dev`. This is helpful when you are building something in the UI and already have the data on your deployed server.
+- Build the entire Coder server + UI locally: `./scripts/develop.sh`. It is useful when you have to contribute with features that are not deployed yet or when you have to work on both, frontend and backend.
+
```

Comment:
These commands are run from two different directories, correct? Making a note of that might help folks.

Reply:
100%